assets/scss/_variables_project.scss

```diff
@@ -1,2 +1,11 @@
 $primary: #326ce5;
 $secondary: #303030;
+
+$display-font-sizes: (
+        1: 3rem,
+        2: 2.5rem,
+        3: 2rem,
+        4: 1.75rem
+);
+
+$display-font-weight: $font-weight-bold;
```

Comment:
Again, I'd vote to align with the Docsy/Bootstrap defaults; even if this means looking slightly different from the pre-upgrade version of the site. WDYT?